x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -628,7 +631,9 @@ export const EditPackagePolicyForm = memo<{
                         onClick={onSubmit}
                         isLoading={formState === 'LOADING'}
                         // Allow to save only if the package policy is upgraded or had been edited
-                        disabled={formState !== 'VALID' || (!isEdited && !isUpgrade)}
+                        disabled={
+                          !hasWritePermissions || formState !== 'VALID' || (!isEdited && !isUpgrade)
+                        }
```

Comment:
Can we also add a tooltip to the save button in the case where `!hasWritePermissions` explaining that they need `All Integrations` to be able to edit these?